packages/react/src/components/Toolbar/Toolbar.tsx

```diff
@@ -184,6 +185,15 @@ class Toolbar extends UIComponent<WithAsProp<ToolbarProps>> {
       return
     }
 
+    if (
+      this.context.target.activeElement === el ||
+      el.contains(this.context.target.activeElement)
+    ) {
+      if (this.containerRef.current) {
+        this.containerRef.current.querySelector('[tabindex]')
```

Comment:
Does it really work? I don't see any logic there 🤔 

Let's also add somehow E2E test for this to avoid issues in future

Reply:
oups